jwst/tests_nightly/general/framework/test_self.py

```diff
@@ -1,18 +1,12 @@
 import pytest
 
 
-pytestmark = [
-    pytest.mark.usefixtures('_jail'),
-    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
-                       reason='requires --bigdata')
-]
-
-
-def test_bigdata(_bigdata):
+@pytest.mark.bigdata
```

Comment:
I feel like this is overkill. This whole file can be deleted.

Reply:
Good catch.